doc/lsst.daf.butler/datastores.rst

```diff
@@ -0,0 +1,79 @@
+.. _daf_butler-datastores:
+
+#######################
+Datastore Configuration
+#######################
+
+A Butler `~lsst.daf.butler.Datastore` is configured in the ``datastore`` section of the top-level Butler YAML configuration.
+The only mandatory entry in the datastore configuration is the ``cls`` key.
+This specifies the fully qualified class name of the Python class implementing the datastore.
+The default is to use the `~lsst.daf.butler.datastores.posixDatastore.PosixDatastore`.
+All other keys depend on the specific datastore class that is selected.
+
+The default configuration values can be inspected at ``$DAF_BUTLER_DIR/config`` and current values can be obtained by calling ``butler config-dump`` on a butler repository.
```

Comment:
Ah, this was the most useful line in the documentation for a "user" like me.  I think this should also be put into the formatter_assembler documentation as well.

Reply:
Ok, although the default configs have now moved and are package resources...